src/lib/POWERMGNT/POWERMGNT.cpp

```diff
@@ -376,6 +394,50 @@ PowerLevels_e POWERMGNT::setPower(PowerLevels_e Power)
         Power = PWR_50mW;
         break;
     }
+#elif defined(TARGET_TX_BETAFPV_900_V1)
+    switch (Power)
+    {
+    case PWR_100mW:
+        Radio.SetOutputPower(0b0000);
+        break;
+    case PWR_250mW:
+        Radio.SetOutputPower(0b0011);
+        break;
+    case PWR_500mW:
+        Radio.SetOutputPower(0b1000);
+        break;
+    case PWR_50mW:
+    default:
+        Power = PWR_100mW;
```

Comment:
This is supposed to be 50mW, not 100, based on the fall though case above `default`